content/source/docs/enterprise/upgrade/differences.html.md

```diff
@@ -0,0 +1,78 @@
+
+Legacy Terraform Enterprise included an event log that logged details about run events. For the new Terraform Enterprise, we replaced it with a [comprehensive audit logging system][audit] that also logs all changes to organization and workspace settings.
+
+The new audit log is available on [private installs](../private/index.html) of Terraform Enterprise.
+
+### `ATLAS_CONFIRM_DESTROY` is now `CONFIRM_DESTROY`
+
+In order to queue a destroy plan on an environment (legacy) or workspace (current), you have to confirm your intentions by setting a special environment variable. The name of that variable has changed to `CONFIRM_DESTROY`, as part of our move away from the Atlas brand name.
+
+## Gone But Returning Soon
+
+### Two-Factor Authentication
+
+The new version of Terraform Enterprise doesn't yet support two-factor authentication in its integrated login system. We plan to add it in a future update.
+
+Depending on your security infrastructure, our [SAML single sign-on support](../saml/index.html) (available on private installs) might also suit your needs.
+
+### Notifications
+
+The new Terraform Enterprise doesn't yet support arbitrary notifications at the end of runs. We plan to add a webhook-based notification system in a future update.
+
+## Permanently Removed
+
+We deliberately removed these features to streamline and improve Terraform Enterprise's core features. Some of these features worked against or distracted from what Terraform Enterprise does best, and some are good ideas that are handled better by external services.
+
+### Packer Integration and Artifact Registry
+
+Terraform Enterprise no longer includes an integrated Packer build system or an artifact registry for built images. Different organizations have such different requirements and expectations for image builds that we believe [running Packer out of your own CI/CD system](https://www.packer.io/guides/packer-on-cicd/index.html) will usually provide a better experience.
+
+### Periodically Queued Plans
+
+In legacy Terraform Enterprise, you could set an environment to periodically queue a plan regardless of whether the configuration had changed. This is gone, because [the run API](../api/run.html) makes it unnecessary.
+
+If you used scheduled runs because Terraform needed to run after events that Terraform Enterprise didn't know about, you can use your CI system to start runs in response to any kind of trigger. If you still want runs to happen on a fixed schedule, you can use any kind of scheduler to make an API call, including a cron job or a Nomad job.
+
+### Individual Collaborators on Environments
+
+Legacy Terraform Enterprise could grant access permissions to individual users; the new Terraform Enterprise handles all workspace permissions via [teams of users](../users-teams-organizations/teams.html).
+
+We removed individual access because it eventually makes your workspace settings a mess. Team-based access is easier to update and govern.
```

Comment:
I might say something like "complicated and difficult to manage" vs a mess, although the unvarnished version is nice too. :)

Reply:
I would also add that you can mimic individual access by creating a team with a single individual. Unless I'm mistaken.